### PR TITLE
Code changes

### PR DESCRIPTION
**Pull request by jvillacampa**
Changes to update to 2019 geography codes and to include 2018 populations

### CODE REVIEW

**Comment by jvillacampa** [comment]
I have added the extraction from the open data platform populations to the code, I am working on the geography ones, which surpringsily enough it might be more difficult.

File: geography_lookup.R [hunk 1]
```diff
@@ -205,37 +216,20 @@ saveRDS(code_dictio, paste0(geo_lookup, 'codedictionary.rds'))
...
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2014),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2015),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2016),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2017),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2018)))
-
-saveRDS(data_simd, paste0(geo_lookup, 'deprivation_geography.rds'))
+# Joining together the extractions of each simd version. 
+data_depr_simd <- rbind(
+  create_simd("a97fca71-ebbb-4897-a611-88024a76ff21", year_list = 1996:2003,
```
Comment (vaelliott) [inline geography_lookup.R:225]: are those horrific looking numbers the resource IDs for the open data files - I'm guessing there is no nice naming convention take makes it easy for someone to scan the code and know they are opening the correct resource file?

Comment (jvillacampa) [reply geography_lookup.R:225]: Yeah, no nice name available. However, these numbers are supposed to be permanent. The url might change, but the id no.

Comment (vaelliott) [reply geography_lookup.R:225]: no body thinks about the analysts having to read the script for the first time or QA these scripts.  At least they won't change...until they do change.

Comment (jvillacampa) [reply geography_lookup.R:225]: Maybe we could add ther url as well, so it makes it easier to check it is the right id?

Comment (vaelliott) [reply geography_lookup.R:225]: I think that might help someone who is trying to figure out how/what the program is doing?

Comment (jvillacampa) [reply geography_lookup.R:225]: Ok, that sounds good. I will add that to both of the lookup scripts

Comment (jvillacampa) [reply geography_lookup.R:225]: Done

**Review by vaelliott — APPROVED** [review]

**Pull request merged**